### PR TITLE
Update opensky to clarify the attributes available in the events

### LINKED ISSUE #28700: Missing attributes in entry event
### Feedback

Hi,
The entry event has 2 attributes that can be interesting and that are not mentioned in the doc: latitude and longitude.
Also, the only attribute that has value in the exit event is callsign: the other ones have a value of 0.
This is worth updating the doc, I think.
Thanks again for your work :)

### URL

https://www.home-assistant.io/integrations/opensky/

### Version

2023.8.4

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by coutadeurf**
## Proposed change

    Updates documentation to clarify the attributes of the events and the defaulting of configuration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- This PR fixes or closes issue: fixes #28700

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/opensky.markdown [hunk 1]
```diff
@@ -34,11 +34,16 @@ altitude:
 - **opensky_entry**: Fired when a flight enters the region.
 - **opensky_exit**: Fired when a flight exits the region.
 
-Both events have four attributes:
+Both events have two attributes in common:
 
 - **sensor**: Name of `opensky` sensor that fired the event.
 - **callsign**: Callsign of the flight.
+
+**opensky_entry** has 4 additionnal attributes:
```
Comment (joostlek) [inline source/_integrations/opensky.markdown]:
```suggestion
**opensky_entry** has 4 additional attributes:
```

**Review by joostlek — CHANGES_REQUESTED** [review]
Typo

File: source/_integrations/opensky.markdown [hunk 2]
```diff
@@ -34,11 +34,16 @@ altitude:
...
+Both events have two attributes in common:
 
 - **sensor**: Name of `opensky` sensor that fired the event.
 - **callsign**: Callsign of the flight.
+
+**opensky_entry** has 4 additional attributes:
+
 - **altitude**: Altitude of the flight in meters.
+- **latitude**: Latitude of the flight in decimal
+- **longitude**: Longitude of the flight in decimal
```
Comment (joostlek) [inline source/_integrations/opensky.markdown]:
```suggestion
- **latitude**: Latitude of the flight in decimal.
- **longitude**: Longitude of the flight in decimal.
```

**Review by joostlek — APPROVED** [review]
If you mark this as ready for review we can merge this :)

**Comment by joostlek** [comment]
Thanks! :)

**Comment by coutadeurf** [comment]
Thank you! First PR for me :)

**Pull request merged**